tests/unit/test_installation.py

```diff
@@ -327,3 +327,33 @@ def test_migrations_broken():
 
     with pytest.raises(IllegalState):
         installation.load(BrokenConfig)
+
+
+def mock_get_status(*args):
+    random_return_value = random.choice(["true", "false"])
+    if args[0] == "enableWorkspaceFilesystem":
+        return {"enableWorkspaceFilesystem": random_return_value}
+
+
+def test_enable_files_in_repos(mocker):
+    ws = create_autospec(WorkspaceClient)
+    ws.current_user.me().user_name = "foo"
+    installation = Installation(ws, "ucx")
+
+    ws.workspace_conf.get_status = mocker.patch(
```

Comment:
don't monkey-patch code you can mock - just use simple mocking `ws.workspace_conf.get_status.return_value = ...`